Examples/Sources/MacroExamples/Playground/main.swift

```diff
@@ -55,3 +55,7 @@ runPeerMacrosPlayground()
 runEnvironmentValueAccessorMacroPlayground()
 
 #endif
+
+// MARK: - SourceLocationMacros
+
+runSourceLoctionMacrosPlayground()
```

Comment:
Would it make sense to also have an `XCTest` test for this that will fail if this breaks instead of just a `print` in the macro examples?

Reply:
Unfortunately XCTest case's expansion (`assertMacroExpansion`) doesn't use this code path.

Reply:
OK, no worries